src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -226,6 +231,63 @@ public static Language ParseSubtitleLanguage(string fileName)
             return Language.Unknown;
         }
 
+        public static (List<string> languageTags, string title, Language language) ParseLanguageTagsAndTitle(string fileName, Episode episode)
```

Comment:
Maybe `ParseSubtitleLanguageInformation`? and return an object with that information.